Examples/Run/Digitization/Common/detail/DigitizationConfigurator.hpp

```diff
@@ -237,6 +250,35 @@ struct DigitizationConfigurator {
           // Set the adapted segmentation class
           dOutputConfig.geometricDigiConfig.segmentation = outputSegmentation;
         }
+
+        // Check for a representing volume configuration, insert if not present
+        Acts::GeometryIdentifier volGeoId =
+            Acts::GeometryIdentifier().setVolume(geoId.volume());
+        auto volRep = volumeLayerComponents.find(volGeoId);
+        if (compactify and volRep != volumeLayerComponents.end()) {
+          // return if the volume representation already covers this one
+          if (dOutputConfig == volRep->second) {
+            return;
+          }
+        } else if (compactify) {
+          volumeLayerComponents[volGeoId] = dOutputConfig;
+          outputDigiComponents->push_back({volGeoId, dOutputConfig});
+        }
```

Comment:
I'd probably write the `if`s the other way round like

```cpp
if(compactify) {
  if(volRep != volumeLayerComponents.end() && dOutputConfig == volRep->second) {
    return;
  }
  // ...
}
```

but not really important.

Reply:
Fixed.